src/calendar/header/index.tsx

```diff
@@ -192,7 +195,18 @@ class CalendarHeader extends Component<CalendarHeaderProps> {
       return customHeaderTitle;
     }
 
-    return (
+    return onPressHeader ? (
+      <TouchableOpacity onPress={() => onPressHeader(month)}>
```

Comment:
First, no need to wrap, you can use Text's `onPress` callback. Second, please don't use inline function calls, this is bad for performance